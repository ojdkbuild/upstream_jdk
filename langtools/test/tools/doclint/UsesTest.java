/*
 * @test /nodynamiccopyright/
 * @bug 8160196
 * @summary Module summary page should display information based on "api" or "detail" mode.
 * @modules jdk.compiler/com.sun.tools.doclint
 * @build DocLintTester
 * @run main DocLintTester -ref UsesTest.out UsesTest.java
 */

/**
 * Invalid use of uses in class documentation.
 *
 * @uses ProvidesTest
 */
public class UsesTest {
    /**
     * Invalid use of uses in field documentation
     *
     * @uses ProvidesTest Test description.
     */
    public int invalid_param;

    /**
     * Invalid use of uses in method documentation
     *
     * @uses ProvidesTest Test description.
     */
    public class InvalidParam { }
}
