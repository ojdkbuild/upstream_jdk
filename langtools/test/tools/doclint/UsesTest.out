UsesTest.java:13: error: invalid use of @uses
 * @uses ProvidesTest
   ^
UsesTest.java:13: error: service-type not found
 * @uses ProvidesTest
   ^
UsesTest.java:13: error: reference not found
 * @uses ProvidesTest
         ^
UsesTest.java:19: error: invalid use of @uses
     * @uses ProvidesTest Test description.
       ^
UsesTest.java:19: error: service-type not found
     * @uses ProvidesTest Test description.
       ^
UsesTest.java:19: error: reference not found
     * @uses ProvidesTest Test description.
             ^
UsesTest.java:26: error: invalid use of @uses
     * @uses ProvidesTest Test description.
       ^
UsesTest.java:26: error: service-type not found
     * @uses ProvidesTest Test description.
       ^
UsesTest.java:26: error: reference not found
     * @uses ProvidesTest Test description.
             ^
9 errors
